### PR TITLE
protoc-gen-go: Fix build command

### PR DESCRIPTION
**Pull request by darklore**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

Fix build command according to installation document. https://github.com/golang/protobuf/blob/master/README.md#installation

### CODE REVIEW

File: Formula/protoc-gen-go.rb [hunk 1]
```diff
@@ -18,9 +19,11 @@ class ProtocGenGo < Formula
   def install
     ENV["GOPATH"] = buildpath
     ENV["GO111MODULE"] = "off"
-    (buildpath/"src/github.com/golang").mkpath
-    ln_s buildpath, buildpath/"src/github.com/golang/protobuf"
-    system "go", "build", "-o", bin/"protoc-gen-go", "protoc-gen-go/main.go"
+    (buildpath/"src/github.com/golang/protobuf").install buildpath.children
+    cd "src/github.com/golang/protobuf/protoc-gen-go" do
+      system "go", "install"
+    end
```
Comment (iporsut) [inline Formula/protoc-gen-go.rb]: I think you needn't `cd` to that path before call `go install`. You can call like this `go install package_path`.

```ruby
system "go", "install", "github.com/golang/protobuf/protoc-gen-go"
```

Comment (darklore) [reply Formula/protoc-gen-go.rb]: Thank you!
I fixed it.

**Review by iporsut — APPROVED** [review]
LGTM.

**Comment by SMillerDev** [comment]
Thanks @darklore

**Pull request closed without merging**